content/en/docs/concepts/storage/volumes.md

```diff
@@ -644,8 +484,8 @@ partition or directory.
 Local volumes can only be used as a statically created PersistentVolume. Dynamic
 provisioning is not supported.
 
-Compared to `hostPath` volumes, `local` volumes are used in a durable and
-portable manner without manually scheduling pods to nodes. The system is aware
+Compared to `hostPath` volumes, local volumes are used in a *durable and
```

Comment:
The original is correct to write `local`, I think. If `hostPath` is in backticks then `local` should be too.
```suggestion
Compared to `hostPath` volumes, `local` volumes are used in a durable and
```